Cargo.toml

```diff
@@ -4,3 +4,49 @@ members = [
 	"primitives/*",
 	"runtime-common",
 ]
+
+[workspace.dependencies]
```

Comment:
Keep the same indentation format as darwinia2.0 repo?

Reply:
Let's keep this format and update the darwinia 2.0 repo.